packages/opentelemetry-exporter-jaeger/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenTelemetry Jaeger Trace Exporter
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+OpenTelemetry Jaeger Trace Exporter allows the user to send collected traces to Jaeger.
+
+[Jaeger](https://jaeger.readthedocs.io/en/latest/), inspired by [Dapper](https://research.google.com/pubs/pub36356.html) and [OpenZipkin](http://zipkin.io/), is a distributed tracing system released as open source by [Uber Technologies](http://uber.github.io/). It is used for monitoring and troubleshooting microservices-based distributed systems, including:
```

Comment:
I'm not sure about the reference:
> and [OpenZipkin](http://zipkin.io/), is a distributed tracing system released as open source by [Uber Technologies](http://uber.github.io/)

Reply:
This is taken from https://www.jaegertracing.io/docs/1.13/#about